### PR TITLE
Copy over installation scripts and Docker config

### LINKED ISSUE #9: Move installation scripts and Docker config over from Animate
The Docker config is used for Goalie and Movement, as well as Animate, so it would make more sense to have it live here.

Linked issue: https://github.com/mesh-adaptation/animate/issues/103.

### PR DESCRIPTION
**Pull request by jwallwork23**
Closes #9.

### CODE REVIEW

**Comment by ddundo** [comment]
@jwallwork23 do I recall correctly that you wanted to copy over something else here? Or is it ready for review as it is?

**Comment by jwallwork23** [comment]
> @jwallwork23 do I recall correctly that you wanted to copy over something else here? Or is it ready for review as it is?

Apologies, done now. Ready for review. Linked PR: https://github.com/mesh-adaptation/animate/pull/104.

File: docker/Dockerfile.firedrake [hunk 1]
```diff
@@ -0,0 +1,48 @@
+FROM firedrakeproject/firedrake-vanilla:2024-02
+
+MAINTAINER Joe Wallwork <joe.wallwork@outlook.com>
```
Comment (ddundo) [inline docker/Dockerfile.firedrake:3]: A warning is raised that `MAINTAINER` is deprecated and that `LABEL` should be used (https://docs.docker.com/reference/dockerfile/#maintainer-deprecated)

**Review by ddundo — APPROVED** [review]
I finally got round to making a fresh install - sorry for being slow. All looks good! Just a minor comment :)

File: docker/Dockerfile.firedrake (hunk 1 above)
Comment (jwallwork23) [reply docker/Dockerfile.firedrake:3]: See #23.

**Pull request merged**